models/notes.py

```diff
@@ -14,6 +14,9 @@ class NotesModel(BaseModel):
     ticketid = db.Column(db.Integer, db.ForeignKey('tickets.id'),
         nullable=False)
 
+    userinfo = db.relationship('UserModel',
+                               backref=db.backref('Notes', lazy=True))
```

Comment:
on line 12 let's rename the `user` field to `user_id` and then let's name this relationship `user`. And `Notes` should be all lowercase.

Reply:
And line 12 could also use a `nullable=False`

Reply:
Just realized why this was bugging me. This relationship doesn't belong here. We still need to change line 12 to `user_id`. However, this relationship should be defined on the UserModel.  See: https://flask-sqlalchemy.palletsprojects.com/en/2.x/models/#one-to-many-relationships for more info.

Reply:
Renamed ```user``` to ```userid``` in the ```NoteModel``` to be consistent with ```ticketid```. 

Moving the relationship to the UserModel makes sense.